src/index.ts

```diff
@@ -153,7 +220,10 @@ const tracker: JupyterFrontEndPlugin<IDebugger> = {
         }
 
         const widget = new MainAreaWidget({
-          content: new Debugger({ connector: state, id })
+          content: new Debugger({
```

Comment:
Let's add the widget to the main area?

For now it's only added to the tracker.

Reply:
I think we could do that. But I don't quite know what we can now display in that widget.

Reply:
Not much for now indeed, but it's still good that it shows up somewhere.

Reply:
Ok. I will change that and try to resolve bug which you found.